enable_logging_API.gs

```diff
@@ -0,0 +1,25 @@
+/**
+* Enable the logging API for cloud project
+* @param {Number} projectNumber of the GCP project
```

Comment:
Declaration of Number object instead of primitive type. Please fix

Reply:
Updated